### PR TITLE
ROX-14935: Remove in-memory bleve from compliance

### PR DESCRIPTION
**Pull request by connorgorman**
## Description

In order to eventually deprecate bleve, we need to remove the in memory use case

## Checklist
- [ ] Investigated and inspected CI test results
- [ ] Unit test and regression tests added
- [ ] Evaluated and added CHANGELOG entry if required
- [ ] Determined and documented upgrade steps
- [ ] Documented user facing changes (create PR based on [openshift/openshift-docs](https://github.com/openshift/openshift-docs) and merge into [rhacs-docs](https://github.com/openshift/openshift-docs/tree/rhacs-docs))

If any of these don't apply, please comment below.

## Testing Performed

CI + manual

### CODE REVIEW

File: central/compliance/standards/index/indexer.go [hunk 1]
```diff
@@ -1,83 +1,30 @@
...
-type controlWrapper struct {
-	*v1.ComplianceControl `json:"control"`
-	Type                  string `json:"type"`
-}
-
-type standardWrapper struct {
-	*v1.ComplianceStandard `json:"standard"`
-	Type                   string `json:"type"`
+type StandardIndexer interface {
+	Search(ctx context.Context, q *v1.Query, opts ...blevesearch.SearchOption) ([]search.Result, error)
```
Comment (dashrews78) [inline central/compliance/standards/index/indexer.go:22]: Do we still need to use `blevesearch` options?  Can we or are we going to move them elsewhere?

**Review by dashrews78 — APPROVED** [review]
LGTM.  Just the one question about search options

**Comment by connorgorman** [comment]
> LGTM. Just the one question about search options

We're going to remove them eventually

**Comment by connorgorman** [comment]
This causes a ton of issues and is a huge web to untangle. Moving to using predicates in memory

**Pull request closed without merging**